client/src/components/Layout/Login/Login.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import "./Login.css";
+
+class Login extends Component {
+  render() {
+    return (
+      <form>
```

Comment:
Should give the form property to make it vertical-centerline

Reply:
@aboodswiti work on this please

Reply:
done